### ISSUE TITLE
bar-color prop not natively reactive for $q.dark

### ISSUE THREAD

**Issue by nothingismagick** [quasarframework/quasar-ui-qactivity#14]
What the issue title says. I would expect that changing the state of `dark` would automatically lighten the CSS of the bar.

**Comment by nothingismagick** [comment]
At the moment I am doing this:
```
          :bar-color="$q.dark.isActive ? 'white' : 'black'"
```

**Comment by hawkeye64** [comment]
QActivity still needs to converted to the new UI kit and has not been updated since the Quasar Dark mode was introduced. It's on my TODO list. :)

**Comment by hawkeye64** [comment]
@nothingismagick This is fixed in v2.0.0-alpha.3
You can keep doing what you're doing, but if you don't specify a `bar-color`, dark mode will be handled automatically.
